admin/views/about.php

```diff
@@ -33,7 +33,7 @@ function wpseo_display_contributors( $contributors ) {
 	<div class="about-text">
```

Comment:
Please also update the plugin name in https://github.com/Yoast/wordpress-seo/blob/stories/CG/textual-suggestions-for-update-page/admin/views/about.php#L31